src/Curator/TeaserableEnricher.php

```diff
@@ -31,23 +31,15 @@ public function enrichWithOrderDate(PbjxEvent $pbjxEvent): void
             return;
         }
 
+        if ($node->has('order_date')) {
+            return;
```

Comment:
@gdbrown i dont think this is right - when a teaser is created it has an order_date, so this check would prevent it from ever updating later on enrich

Reply:
nm

Reply:
@efjacobson it might not though, this really just needs to ensure it has one.  ref https://github.com/triniti/curator-php/blob/master/src/Enricher/TeaserableEnricher.php